templates/geth/README.md

```diff
@@ -0,0 +1,37 @@
+# template: github.com/threefoldfoundation/tfchain-templates/geth/0.0.2
+
+## Description
+
+This template is responsible for deploying a geth.
+
+## Schema
+
+- `network`: Ethereum network (testnet or rinkeby) (default testnet)
```

Comment:
we will also need to deploy a full node for mainnet eventually, is this possible or is this just not documented?

Reply:
For deploying a node on mainnet just provide an (`""`) for the network parameter.